ml_peg/app/utils/build_components.py

```diff
@@ -368,8 +405,9 @@ def build_test_layout(
     metric_weights = build_weight_components(
         header="Metric Weights",
         table=table,
-        use_thresholds=True,
+        use_thresholds=thresholds is not None,
```

Comment:
These are for the metric tables, so should always have thresholds, shouldn't they?